extensions/amp-iframe/amp-iframe.md

```diff
@@ -208,3 +208,12 @@ Iframes are identified as tracking/analytics iframes if they appear to serve no
 ## Validation
 
 See [amp-iframe rules](https://github.com/ampproject/amphtml/blob/master/extensions/amp-iframe/0.1/validator-amp-iframe.protoascii) in the AMP validator specification.
+
+## Guideline: prefer [specific AMP components](https://github.com/ampproject/amphtml/tree/master/extensions) to `amp-iframe`
```

Comment:
When this goes into the docs site, please check that the link there points to the right destination. CC @pbakaus